app/presenters/oauth_metadata_presenter.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class OauthMetadataPresenter < ActiveModelSerializers::Model
+  include RoutingHelper
+
+  attributes :issuer, :authorization_endpoint, :token_endpoint,
+             :registration_endpoint, :revocation_endpoint, :scopes_supported,
+             :response_types_supported, :response_modes_supported,
+             :grant_types_supported, :token_endpoint_auth_methods_supported
+
+  def issuer
+    "http#{Rails.configuration.x.use_https ? 's' : ''}://#{Rails.configuration.x.web_domain}/"
```

Comment:
I think I can just replace this with:
```suggestion
    root_url
```

But in other places we've this line: https://github.com/mastodon/mastodon/blob/main/lib/mastodon/version.rb#L68

Could that just be `root_url` too?

Reply:
Replaced with root_url.